docs/ruleset_functions/get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM.md

```diff
@@ -0,0 +1,140 @@
+
+**Inputs:**  
+- **B-RMI,P-RMI**: To calculate the supply, return, exhaust, and relief total fan power, CFM, quantity, and to determine the consistency of pressure drop across fans (if there are multiple associated with a specific category (supply, return, exhaust, relief) for a fan system object sent to this function.   
+- **fan_system_obj**: The fan system object sent to this function.  
+
+**Returns:**  
+- **get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM**: The function calculates and returns the supply, return, exhaust, and relief total fan power, CFM, quantity, and information about whether the pressure drop is consistent across the fans if more than one for a fan system object.   The function returns a dictionary that saves the supply, return, exhaust, and relief fan power as a list, saves the supply, return, exhaust, and relief cfm as a list, saves the supply, return, exhaust, and relief quantity as a list, and saves for each fan whether the pressure drop is undefined, identical, or different across fans (if only one it will return undefined or identical) {"Fan_Power": [supply fan power kW, return fan power kW, exhaust fan power kW, relief fan power kW], "Fan_CFM": [supply fan power CFM, return fan power CFM, exhaust fan power CFM, relief fan power CFM], "Fan_Qty": [supply fan quantity, return fan quantity, exhaust fan quantity, relief fan quantity], "Fan_Pressure": [returns either "IDENTICAL", "UNDEFINED", "DIFFERENT" for supply fans, returns either "IDENTICAL", "UNDEFINED", "DIFFERENT" for return fans, returns either "IDENTICAL", "UNDEFINED", "DIFFERENT" for exhaust fans, returns either "IDENTICAL", "UNDEFINED", "DIFFERENT" for relief fans]}. Values will be equal to zero where not defined for a fan system or as otherwise specified above. For CFM values this function assumes that all fans associated with the supply_fans object are in parallel (i.e., if multiple fans the cfm is additive.) but returns the quantity and information about the pressure drop across fans to help assess whether series or parallel.
+ 
+**Function Call:**  
+1. get_fan_object_electric_power 
+
+
+## Logic:    
+Supply:
+- Reset total fan power to zero: `total_fan_power = 0`  
+- Reset total cfm to zero: `total_fan_cfm = 0`  
+- Reset fan_qty equal to 0 (counts the quantity of fans): `fan_qty = 0`  
+- Reset pressure drop identical across fans: `pressure_drop_identical = true`  
+- Reset pressure drop defined for all fans: `pressure_drop_defined_for_all_fans = true`  
+- Check if there are any fan objects associated with the fan system's supply fans:`if len(fan_system_obj.supply_fans) != 0 and fan_system_obj.supply_fans != Null:`   
+  - Loop through the supply fan: `for fan_obj in fan_system_obj.supply_fans:`  
+    - Set the fan_elec_power equal to zero: `fan_elec_power = 0`
+    - Set the fan_cfm equal to zero: `fan_cfm = 0`
+    - Get the fan objects total kW: `fan_elec_power = get_fan_object_electric_power(P_RMI, fan_obj)`  
```

Comment:
Why is it referencing P_RMI?

Reply:
@mkarpman Revised to make it non specific. Thanks.